config-xml/addon/pom.xml

```diff
@@ -50,6 +50,17 @@
     </dependencies>
 
     <build>
+        <resources>
+            <resource>
+                <directory>../schema/</directory>
+                <includes>
+                    <include>windup-ruleset-jboss-${project.version}.xsd</include>
```

Comment:
I am keeping non-versioned, a copy just for the sake of the version being in the file name is not needed, at least not now.

Later, we could perhaps externalize the schema version to root POM and use ${...} inside of the schema, with resources filtering; then, copying + filtering may be useful.